core-library/component-styling/input-component/_input-component.scss

```diff
@@ -22,36 +22,106 @@
 
 @mixin layout {
 
-.input-container {
-  user-select: none;
+  .input-container {
+    user-select: none;
+  
+    .input-text {
+      width: 100%;
+      display: grid;
+      position: relative;
+      
+      .input-label {
+        display: flex;
+        font-size: 16px;
+        font-weight: 700;
+      }
+      .input-desc {
+        font-size: 14px;
+        margin: 4px 0px 0px 0px;
+      }
 
-  .input-text {
-    width: 100%;
-    display: grid;
-    position: relative;
-  }
+      .input-hint {
+        font-size: 14px;
+        color: #6F6F71;
+        margin: 8px 0px 0px 0px;
+      }
+      .required-star {
+        font-size: 12px;
+        color: var(--critical-text);
+        font-weight: 400;
+        line-height: 16px;
+        justify-self:flex-start;
+        margin-right: 4px;
+      }
+      
 
+      @include size.selector(large) {
 
-.passwordIcon {
-  justify-self: flex-end;
-  align-self: flex-end;
-  //TODO: Calculate the position of the eye icon
-}
+        .input-label {
+          font-size: 20px;
+          font-weight: 700;
+        }
+
+        .input-desc {
+          font-size: 16px;
+          margin: 4px 0px 0px 0px;
+        }
 
-.transparentButton {
-  position: absolute;
-  justify-self: flex-end;
-  align-self: flex-end;
-  margin-bottom: 0px;
-  background: transparent;
-  z-index: 1;
+        .input-hint {
+          font-size: 16px;
+          color: #6F6F71;
+          margin: 12px 0px 0px 0px;
+        }
 
-  &:focus{
-    box-shadow: 0 0 0 3px var(--focus-border);
+        .required-star {
+          font-size: 16px;
+          color: var(--critical-text);
+          font-weight: 400;
+          line-height: 16px;
+          justify-self: flex-start;
+          margin-right: 5px;
+        }
+      }
+    }
   }
-  
-}
 
-}
-   
+
+  // .passwordIcon {
```

Comment:
since password Icon styling is resolved we can remove this comment too - thanks!